certified-connectors/Insiders Cloud/apiDefinition.swagger.json

```diff
@@ -0,0 +1,441 @@
+        "Original",
+        "Other"
+      ]
+    },
+    "Field": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "value": {},
+        "type": {
+          "$ref": "#/definitions/FieldType"
+        },
+        "geo": {
+          "$ref": "#/definitions/Geometry"
+        },
+        "verified": {
+          "type": "boolean"
+        },
+        "visible": {
+          "type": "boolean"
+        },
+        "alteredBy": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "verifierAction": {
+          "$ref": "#/definitions/FieldVerifierAction"
+        },
+        "checkState": {
+          "$ref": "#/definitions/FieldCheckState"
+        }
+      }
+    },
+    "FieldType": {
+      "type": "string",
+      "description": "",
+      "enum": [
+        "Bool",
+        "Date",
+        "String",
+        "Number"
+      ]
+    },
+    "Geometry": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "top": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "left": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "width": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "height": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "pageIndex": {
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "FieldVerifierAction": {
+      "x-nullable": true,
+      "type": "string",
+      "description": "",
+      "enum": [
+        "NoAction",
+        "Imported",
+        "Confirmed",
+        "Changed",
+        "DBLookup",
+        "AutoFill",
+        "OCR"
+      ]
+    },
+    "FieldCheckState": {
+      "x-nullable": true,
+      "type": "string",
+      "description": "",
+      "enum": [
+        "OK",
+        "Overridden",
+        "MandatoryOverridden",
+        "Disabled"
+      ]
+    },
+    "Table": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "rows": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Row"
+          }
+        },
+        "columns": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Row": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "pageIndex": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "cells": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Field"
+          }
+        }
+      }
+    },
+    "DocumentState": {
+      "x-nullable": true,
+      "type": "string",
+      "description": "",
```

Comment:
I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).